src/apps/lwaftr/lwutil.lua

```diff
@@ -49,3 +51,16 @@ function keys(t)
    return result
 end
 
+
+-- Local bindings for constants that are used in the hot path of the
```

Comment:
I'm not quite understanding this review comment. He'll still need to cache relative offsets to use with pointers if he does that, so where's the conflict?